docs/superpowers/plans/2026-07-24-identity-marketplace-seller-web-only.md

```
@@ -0,0 +1,209 @@
+```ts
+// Seller-side identity actions (list a handle for sale, accept/reject an offer)
+// are web-only — the tiny.place backend exposes no seller routes and the
+// vendored SDK is a buyer-side compatibility wrapper (see #4920). We point
+// sellers at the web app instead. Hardcoded prod URL, matching the
+// `FUND_PAGE_URL` precedent in X402ConfirmDialog.tsx.
+const SELL_ON_WEB_URL = 'https://tiny.place/identities';
+```
+
+- [ ] **Step 5: Insert the note JSX in `TradingTab`**
+
+In the same file, insert the following block immediately **before** line 996 (`    </div>` that closes the outer `<div className="space-y-4">`), i.e. after the Recent Sales `</div>` at line 995:
```

markdownlint-cli2 (0.23.0)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)